vendor/composer/autoload_namespaces.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
vendor dir should not have been commited

Reply:
whoops, so far for automation :-)